tests/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 # nnstreamer-edge test
+SET(CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -DENABLE_MQTT=1")
```

Comment:
let's move this to root make file, also need fix this.
`SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DENABLE_MQTT=1")`

Reply:
Updated. Thanks.